prisma/schema.prisma

```diff
@@ -7,11 +7,62 @@ generator client {
   provider = "prisma-client-js"
 }
 
-model User {
-  username String @id
-  name     String
-  email    String @unique
+model Users {
+  username  String @id
+  firstName String
+  lastName  String
+  email     String @unique
 
-  createdAt DateTime @default(now())
-  updatedAt DateTime @updatedAt
+  createdAt    DateTime       @default(now())
+  updatedAt    DateTime       @updatedAt
+  subdomians   Subdomains[]
+  certificates Certificates?
+}
+
+model Subdomains {
+  id          Int             @id @default(autoincrement())
+  username    String
+  name        String
+  type        SubdomainType
+  value       String
+  description String
+  course      String
+  ports       String
+  createdAt   DateTime        @default(now())
+  updatedAt   DateTime        @updatedAt
+  expiresAt   DateTime
+  status      SubdomainStatus
+  user        Users           @relation(fields: [username], references: [username])
+}
+
+model Certificates {
+  id          Int          @id @default(autoincrement())
+  username    String       @unique
+  subject     String
+  certificate String
+  privkey     String
```

Comment:
Let's file a follow-up to figure out the encryption strategy we want to use for this (e.g., in Prisma, in MySQL, or in TS).